index.html

```diff
@@ -73,7 +77,7 @@
 
   <div id="map"></div>
 
-  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" src="js/script.js"></script>
+  <script data-spreadsheet="0Agcr__L1I1PDdEpoMnhxR0RHdkFsWlFtNTlEZlltR0E" data-actionkit="peoples-climate-plan-actions|break-free-world" src="js/script.js"></script>

```

Comment:
Hmm.  Let's just remove this for now and not show any on the default map quite yet.